neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -146,25 +151,40 @@ def upload_experiment_source(self, experiment, data, progress_indicator):
         # TODO: handle `FileChunkStream` or update `neptune.experiments.Experiment._start`
         pass
 
-    @with_api_exceptions_handler
-    def create_system_channel(self, experiment, name, channel_type):
-        channel_id = f'{alpha_consts.MONITORING_ATTRIBUTE_SPACE}{name}'
-        dummy_log_string = alpha_operation.LogStrings(
+    def _create_channel(self, experiment: Experiment, channel_id: str, channel_type: str):
+        """This function is responsible for creating 'fake' channels in alpha projects.
+
+        Since channels are abandoned in alpha api, we're mocking them using empty logging operation."""
+
+        if channel_type == ChannelType.TEXT.value:
+            operation = alpha_operation.LogStrings
+        elif channel_type == ChannelType.NUMERIC.value:
+            operation = alpha_operation.LogFloats
+        elif channel_type == ChannelType.IMAGE.value:
+            operation = alpha_operation.LogFloats
+        else:
+            raise NeptuneException(f"We're not supporting {channel_type} channel type.")
+
+        log_empty_operation = operation(
             path=alpha_path_utils.parse_path(channel_id),
             values=[],
         )  # this operation is used to create empty attribute
         self._execute_alpha_operation(
             experiment=experiment,
-            operations=[dummy_log_string],
+            operations=[log_empty_operation],
         )
-        system_channels = self.get_system_channels(experiment)
-        for channel in system_channels:
-            if channel.name == name:
-                return channel
-        raise ChannelNotFound(channel_id=channel_id)
 
     @with_api_exceptions_handler
-    def get_system_channels(self, experiment):
+    def create_channel(self, experiment, name, channel_type) -> ChannelWithLastValue:
+        channel_id = f'{alpha_consts.LOG_ATTRIBUTE_SPACE}{name}'
+        self._create_channel(experiment, channel_id, channel_type)
+        try:
+            channel = self.get_channels(experiment)[name]  # TODO: implement this
```

Comment:
1. Good idea.
2. It looks like [this discussion](https://github.com/neptune-ai/neptune-client/pull/371/files#r561777573) gained some arguments for. I'll assume that attribute was created if there's no error in run operations
3. Good catch. Since we're decided to solution 2. the problem no longer exists. But such double wrapping with `_execute_alpha_operation` exists in case of `_execute_alpha_operation`. I've created an issue for that.